packages/bot-skeleton/src/services/tradeEngine/utils/helpers.js

```diff
@@ -54,7 +54,13 @@ export const getDirection = ticks => {
     return direction;
 };
 
-export const getLastDigit = tick => (Number.isInteger(tick) ? tick % 10 : Number(tick.toString().slice(-1)));
+export const getLastDigit = tick => {
+    let number_string = tick;
```

Comment:
Are there any reason to re-declare this instead of just using `tick` ?

Reply:
<img width="624" alt="image" src="https://user-images.githubusercontent.com/97010868/158958636-f28bb50e-0d76-4b64-8b9c-2cb102831ec6.png">

thanks for asking 🙇 
I was getting this lint error so I had to declare a new variable for explicit type conversion.